com.unity.ml-agents/Runtime/Demonstrations/DemonstrationRecorder.cs

```diff
@@ -32,6 +32,14 @@ public class DemonstrationRecorder : MonoBehaviour
         [Tooltip("Whether or not to record demonstrations.")]
         public bool Record;
 
+        /// <summary>
+        /// Number of steps to record. If this number is higher than zero, the editor will stop
+        /// playing when it reaches this threshold.
+        /// </summary>
+        [Tooltip("Number of steps to record. If this number is higher than zero, the editor will stop " +
```

Comment:
Maybe change to "Number of steps to record. The editor will stop playing when it reaches this threshold. Set to zero to record indefinitely."

Reply:
updated to use this language ^